src/main/java/com/multithreading/App.java

```diff
@@ -1,11 +1,21 @@
 package com.multithreading;
 
-/**
- * Hello world!
- *
- */
+import java.util.Scanner;
+
 public class App {
     public static void main(String[] args) {
-        System.out.println("Hello World!");
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Enter a string: ");
+        String input = scanner.nextLine();
+        scanner.close();
+
+        TextProcessor processor = new TextProcessor();
+
+        try {
+            System.out.println("Capitalized String: " + processor.process(input));
```

Comment:
nit: recommend splitting the debut output from the execution

String result = process(input);
print("Capitalized: "+result);